ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Docker/tasks/main.yml

```diff
@@ -187,55 +187,66 @@
     - ansible_distribution == "Ubuntu"
   tags: docker
 
-- name: Install Docker for ALL
+- name: Install Docker for Ubuntu
   package: "name=docker-ce state=latest"
   when:
     - docker_installed.rc != 0
-    - (ansible_distribution == "RedHat" and ansible_distribution_major_version == "7" ) or (ansible_distribution == "Ubuntu") or (ansible_distribution == "CentOS" and ansible_distribution_major_version == "7" )
+    - ansible_distribution == "Ubuntu"
     - (ansible_architecture == "x86_64" or ansible_architecture == "s390x" or ansible_architecture == "ppc64le")
   tags:
     - docker
     # TODO: Package installs should not use latest
     - skip_ansible_lint
 
-- name: Add Jenkins user to the docker group for RHEL7, cent7, and ubuntu
+- name: Install Docker for Cent/Rhel 7
```

Comment:
Is It really CentOS 7 or are we conflating that it's Ansible 7?

Reply:
I'm not sure, I just used the same syntax as what was in the file before I made my changes

Reply:
This is ok @karianna - `ansible_distribution_version` is checking the version of the OS distribution on the target systemm, not the ansible version (Compare with https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/573/files which is the ansible version check)